frontend/src/v5/ui/routes/dashboard/projects/drawings/drawingDialogs/editDrawingDialog.component.tsx

```diff
@@ -74,54 +72,12 @@ export const EditDrawingDialog = ({ open, onClickClose, drawing }:Props) => {
 			confirmLabel={formatMessage({ id: 'drawings.edit.ok', defaultMessage: 'Save Drawing' })}
 			maxWidth="sm"
 			{...formState}
-			isValid={dirtyValuesChanged(formData, drawing) && formState.isValid}
+			isValid={dirtyValuesChanged(formData, drawingId) && formState.isValid}
 		>
-			<SectionTitle>
-				<FormattedMessage
-					id="drawings.edit.informationTitle"
-					defaultMessage="Drawing information"
-				/>
-			</SectionTitle>
-			<ShareTextField
-				label={formatMessage({ id: 'drawings.id', defaultMessage: 'ID' })}
-				value={drawing._id}
-			/>
-			<FormTextField
-				control={control}
-				name="name"
-				label={formatMessage({ id: 'drawings.creation.form.name', defaultMessage: 'Name' })}
-				formError={errors.name}
-				disabled={!isProjectAdmin}
-				required
-			/>
-
-			<FormTextField
-				control={control}
-				name="number"
-				label={formatMessage({ id: 'drawings.creation.form.number', defaultMessage: 'Drawing Number' })}
-				formError={errors.number}
-				disabled={!isProjectAdmin}
-				required
-			/>
-			<FormSelect
-				control={control}
-				name="type"
-				label={formatMessage({ id: 'drawings.creation.form.type', defaultMessage: 'Category' })}
-				disabled={!isProjectAdmin}
-				required
-			>
-				{types.map((type) => (
-					<MenuItem key={type} value={type}> {type}</MenuItem>
-				))}
-			</FormSelect>
-			<FormTextField
-				control={control}
-				name="desc"
-				label={formatMessage({ id: 'drawings.creation.form.description', defaultMessage: 'Description' })}
-				formError={errors.desc}
-				disabled={!isProjectAdmin}
-			/>
-			<UnhandledErrorInterceptor expectedErrorValidators={[nameAlreadyExists, numberAlreadyExists]} />
+			{drawing.calibration.units
```

Comment:
Does selectAreStatsPending not work for this?

Reply:
No because settings are not part of the stats. You may already have the stats fetched, but the calibration data might be missing